### PR TITLE
Add placeholder localized resx files 

### PR DESCRIPTION
**Pull request by kythant**
Add placeholder localized resx files to prepare for localization.

TDBuild will eventually replace these files with the localized resx.

The resources will fall back to the originals in VSPackage.resx if it cannot find it in the culture specific resx.

### CODE REVIEW

File: dev/VSIX/Extension/Cs/Dev16/WindowsAppSDK.Cs.Extension.Dev16.csproj [hunk 1]
```diff
@@ -163,6 +164,62 @@
       <Link>Resources\VSPackage.resx</Link>
       <Generator></Generator>
     </EmbeddedResource>
+    <EmbeddedResource Include="$(ExtensionDir)Cs\Common\cs-CZ\VSPackage.cs-CZ.resx">
```
Comment (huichen123) [inline dev/VSIX/Extension/Cs/Dev16/WindowsAppSDK.Cs.Extension.Dev16.csproj:167]: who decides these 14 languages? @axelandrejs

Comment (kythant) [reply dev/VSIX/Extension/Cs/Dev16/WindowsAppSDK.Cs.Extension.Dev16.csproj:167]: This are the languages visual studio is localized to.

**Review by huichen123 — APPROVED** [review]

**Review by BenJKuhn — APPROVED** [review]

**Comment by DefaultRyan** [comment]
Starting to get a bit of duplication between the dev16 and dev17 projects, so would be nice to combine those assets into shared projitems, one each for cpp and cs. Not blocking on this one, but something for us to try and do next time we're touching anything here.

**Review by DefaultRyan — APPROVED** [review]

**Pull request merged**